themes/bootstrap3/templates/RecordTab/staffviewarray.phtml

```diff
@@ -8,9 +8,12 @@
     <tr>
       <th><?=$this->escapeHtml($field)?></th>
       <td>
-        <?php foreach ((array)$values as $value): ?>
-          <?=$this->escapeHtml(is_array($value) ? print_r($value, true) : $value)?><br />
-        <?php endforeach; ?>
+        <?php
+          if (is_array($values) && count($values) == 1 && is_int(array_key_first($values))) {
+            $values = array_shift($values);
+          }
+        ?>
```

Comment:
Do we need this special array_shift case anymore? It looks to me like with your latest changes, a flat value and a single-valued array actually display the same -- i.e.

```
        <?=$this->printArrayHtml("Foo")?>
```

and

```
        <?=$this->printArrayHtml(["Foo"])?>
```

look exactly the same to me.

Reply:
Quite true! It's not needed anymore as that case is handled within the helper now. It's been corrected; I also added passing it a string value into the unit tests.